MMA Removal Utility/SingleTenantSetupInstallation.md

```diff
@@ -0,0 +1,302 @@
+# **Steps to install single-tenant AzTS MMA Removal Utility**
+
+In this section, we will walk you through the steps of setting up AzTS MMA Removal Utility. This setup can take up to 30 minutes.
+
+Before initiating the setup please validate the prerequisites, download and extract deployment package using [Prerequisites](./Prerequisites.md).
+
+There are two methods to setup AzTS MMA Removal Utility solution:
+
+- [**Method A:**](#method-a) This method provides granular level details of the different steps involved in setting up AzTS MMA Removal Utility. Setup process is divided into five steps and seprate command is provided for each step.
+
+- [**Method B:**](#method-b) Provides a quick way to install AzTS MMA Removal Utility solution by wrapping up multiple steps into a single consolidated command. 
```

Comment:
Provides a faster way to install AzTS MMA Removal Utility solution by wrapping up multiple steps into a single (consolidated) command.

Reply:
Addressed.